Framework/Algorithms/src/MaxEnt.cpp

```diff
@@ -204,6 +204,29 @@ void MaxEnt::init() {
                                                          500, mustBePositive,
                                                          Direction::Input),
                   "Maximum number of iterations in alpha chop.");
+  declareProperty(
+      make_unique<WorkspaceProperty<>>(
+          "DataLinearAdj", "", Direction::Input, PropertyMode::Optional,
+          boost::make_shared<EqualBinSizesValidator>(errorLevel, warningLevel)),
+      "Adjusts the calculated data by multiplying each value by the "
+      "corresponding Y value of this workspace. "
+      "The data in this workspace is complex in the same manner as complex "
+      "input data.");
+  declareProperty(
+      make_unique<WorkspaceProperty<>>(
+          "DataConstAdj", "", Direction::Input, PropertyMode::Optional,
+          boost::make_shared<EqualBinSizesValidator>(errorLevel, warningLevel)),
+      "Adjusts the calculated data by adding to each value the corresponding Y "
+      "value of this workspace. "
+      "If DataLinearAdj is also specified, this addition is done after its "
+      "multiplication. "
+      "The data in this workspace is complex in the same manner as complex "
+      "input data.");
+  declareProperty(
+      "PerSpectrumImage", true,
```

Comment:
As discussed please change the name of this property to PerSpectrumReconstruction